intera_examples/scripts/go_to_cartesian_pose.py

```diff
@@ -86,6 +86,9 @@ def main():
     parser.add_argument(
         "--rotational_accel", type=float, default=1.57,
         help="The max rotational acceleration of the endpoint (rad/s/s)")
+    parser.add_argument(
+        "--timeout", type=float, default=None,
+        help="Max time in seconds to block before returning")
```

Comment:
This help message should be more clear to indicate what happens when the timeout is reached. It should also state the behavior when `none` is passed (no timeout).